site/pages/speakers.en.rst

```diff
@@ -1,33 +1,91 @@
 
 .. title: Speakers
 .. slug: speakers
-.. date: 2019-04-25 15:06:26 UTC+07:00
+.. date: 2019-05-16 19:14:38 UTC+07:00
 .. tags:
 .. category:
 .. link:
 .. description: List of confirmed speakers.
-.. type: macro
+.. type: text
 
 
 
 .. class:: clearfix
 
 
+Alex Khaerov
+============
 
-Understanding and Implementing Generative Adversarial Networks (GANs): One of the BIGGEST Breakthroughs in the Deep Learning Revolution
-=======================================================================================================================================
 
 
+.. image:: https://secure.gravatar.com/avatar/320f3b558c773592bba16c976d1b28d1?s=500
+    :alt: Alex Khaerov
+    :height: 200px
+    :align: right
+    :class: img-circle img-responsive
 
-Format: Talk (~30-45 minutes)
 
-Abstract
---------
 
-Generative Adversarial Networks are one of the latest advancements in Deep Learning. Interested in knowing how to generate content (images, music, and much more) instead of classifying one into categories? Let's dive into the details of GANs: One of the BIGGEST Breakthroughs in the DL Revolution.
 
-Bio
----
+
+.. class:: fa fa-twitter fa-fw
+
+    `hayorov <https://twitter.com/hayorov>`_
+
+
+
+
+
+Talk: Kill the mutants, protect your code!
+
+Biography
+---------
+
+**Alex Khaerov** is a development lead at
+[Chainstack](https://chainstack.com), with 9+ years in web services
```

Comment:
Hmm now this page is a mix of Markdown (links) and RestructuredText (headings). 
I'm guessing it would be the least effort to switch everything to Markdown (change the filename to `.md`), since the descriptions are coming straight from papercall, right? 
I don't think we need to wait for this, if I find some time I'll do it later.